test_schemas/en/test_new_routing_answered_unanswered.json

```diff
@@ -0,0 +1,221 @@
+{
+    "mime_type": "application/json/ons/eq",
+    "language": "en",
+    "schema_version": "0.0.2",
+    "data_version": "0.0.3",
+    "survey_id": "0",
+    "title": "Routing answered/unanswered",
+    "theme": "default",
+    "description": "A questionnaire to demo routing when question answered/unanswered",
+    "metadata": [
+        {
+            "name": "user_id",
+            "type": "string"
+        },
+        {
+            "name": "period_id",
+            "type": "string"
+        },
+        {
+            "name": "ru_name",
+            "type": "string"
+        }
+    ],
+    "questionnaire_flow": {
+        "type": "Linear",
+        "options": {
+            "summary": {
+                "collapsible": false
+            }
+        }
+    },
+    "sections": [
+        {
+            "id": "default-section",
+            "groups": [
+                {
+                    "blocks": [
+                        {
+                            "type": "Question",
+                            "id": "block-1",
+                            "question": {
```

Comment:
Can we add some guidance, something we did when we added new routing previously, something like: 
```
"guidance": {
    "contents": [
        {
            "description": "Pick an answer to route to the “answered” page otherwise you will be routed to the “unanswered” page"
        }
    ]
},
```

Reply:
Added 👍